code/modules/clothing/head/helmet.dm

```diff
@@ -75,9 +75,23 @@
 	desc = "A bulletproof helmet that excels in protecting the wearer against traditional projectile weaponry and explosives to a minor extent."
 	icon_state = "swat"
 	item_state = "swat-alt"
-	armor = list(MELEE = 15, BULLET = 60, LASER = 10, ENERGY = 10, BOMB = 40, BIO = 0, RAD = 0, FIRE = 50, ACID = 50)
+	armor = list(MELEE = 15, BULLET = 50, LASER = 10, ENERGY = 10, BOMB = 40, BIO = 0, RAD = 0, FIRE = 50, ACID = 50)
 	dog_fashion = null
 
+/obj/item/clothing/head/helmet/laserproof
+	name = "ablative helmet"
+	desc = "A helmet that excels in protecting the wearer against energy projectiles. Projects a smaller energy field around the user, allowing a chance of energy projectile deflection no matter where on the user it would hit."
+	icon_state = "helmet_reflect"
+	item_state = "helmet_reflect"
+	armor = list(MELEE = 10, BULLET = 10, LASER = 50, ENERGY = 50, BOMB = 0, BIO = 0, RAD = 0, FIRE = 100, ACID = 100)
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	var/hit_reflect_chance = 10
+
+/obj/item/clothing/head/helmet/laserproof/IsReflect()
+	var/mob/living/carbon/human/user = loc
+	if(prob(hit_reflect_chance) && (user.head == src))
+		return 1
```

Comment:
this actually reduces the chance for ablative to reflect things. as its now 10% chance, then a 30% chance. probabilities don't add, so its now a 37% chance to deflect instead of 40%.

Reply:
I don't understand how a 10% + a 30% chance doesn't stack to equal 40%. Can you explain this to me like I am a complete idiot?

Reply:
Got it. changed to 33%

Reply:
Because you add consecutive probabilities together, you have to multiply them.

    Probability of rolling a 1: 1/6
    Probability of rolling a 6: 1/6

So the probability of rolling a 1 or a 6 is 1/6 + 1/6 = 2/6 = 1/3.

The probability of both events happening together on the same die is zero, at least with a single throw. But if you wanted to know the probability of rolling a 1 and then rolling a 6, that’s when you would multiply (the probability would be 1/6 * 1/6 = 1/36).